src/cameras/frame_grabber.py

```diff
@@ -62,4 +82,14 @@ def _start_frame_loop(self, save_video=False):
         else:
             logger.info("Frame loop thread exited.")
         finally:
-            self._writer.release()
+            options = SaveOptions(
+                writer_dir=Path().joinpath(
+                    self.session_writer_path,
+                    "raw_frame_capture",
+                    f"webcam_{self._webcam_id}",
+                ),
+                fps=self.current_fps,
+                frame_width=self._frame_width,
+                frame_height=self._frame_height,
+            )
+            video_writer.save(options)
```

Comment:
The framethread that the OpenCV Camera relies on saves the raw frames we get to data folder